public/500.html

```diff
@@ -1,66 +1,81 @@
 <!DOCTYPE html>
-<html>
+<html class="govuk-template">
 <head>
-  <title>We're sorry, but something went wrong (500)</title>
-  <meta name="viewport" content="width=device-width,initial-scale=1">
-  <style>
-  .rails-default-error-page {
-    background-color: #EFEFEF;
-    color: #2E2F30;
-    text-align: center;
-    font-family: arial, sans-serif;
-    margin: 0;
-  }
-
-  .rails-default-error-page div.dialog {
-    width: 95%;
-    max-width: 33em;
-    margin: 4em auto 0;
-  }
+  <meta charset="utf-8" />
+  <title>Page not found – GOV.UK</title>
+  <link rel="stylesheet" media="all" href="/errors.css" />
+</head>
 
-  .rails-default-error-page div.dialog > div {
-    border: 1px solid #CCC;
-    border-right-color: #999;
-    border-left-color: #999;
-    border-bottom-color: #BBB;
-    border-top: #B00100 solid 4px;
-    border-top-left-radius: 9px;
-    border-top-right-radius: 9px;
-    background-color: white;
-    padding: 7px 12% 0;
-    box-shadow: 0 3px 8px rgba(50, 50, 50, 0.17);
-  }
+<body class='govuk-template__body'>
```

Comment:
I think this also needs a 'skip to content' link:

```suggestion
<body class='govuk-template__body'>
  <a href="#content" class="govuk-skip-link">Skip to main content</a>
```